baseplate/clients/sqlalchemy.py

```diff
@@ -226,6 +226,7 @@ def on_before_execute(
 
         trace_name = f"{context_name}.execute"
         span = server_span.make_child(trace_name)
+        span.set_tag("protocol", "sqlalchemy")
```

Comment:
sqlalchemy isn't a protocol, it's a package providing an API to interface python=>SQL, sqlalchemy is just an implementation detail.

Reply:
Technically neither is sql :P I think "protocol" is a misnomer regardless since the coupling between this class and the prometheus observer is tight: we can't expect any other class implementing "sql" or "database" to expose the same tags as the sqlalchemy client. This supports your idea below of getting rid of this tag.

Reply:
Fair enough. What about if we're targeting a slow database and want different latency buckets?

Reply:
The "protocol" here doesn't change the answer to your question, but I think you answered it in an earlier comment: the combination of statement and db/db_host would allow us to identify slower queries.